docs/topics/rn-resolved-issues.adoc

```diff
@@ -16,15 +16,15 @@ For a complete list of all issues resolved in this release, see the list of link
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-3082[WINDUP-3082]
-|Eclipse plugin
-|A `NullPointerException` error occurs when browsing rules in the Rulesets view.
+|link:https://issues.redhat.com/browse/WINDUP-3125[WINDUP-3125]
+|ItelliJ IDEA plugin and Visual Studio Code extension
+|The *Apply QuickFix* menu option is still active after a QuickFix is applied.
```

Comment:
I think this would be clearer: `The *Apply QuickFix* menu option is not disabled after a QuickFix is applied.`